uvm/impl/com/untangle/uvm/SettingsManagerImpl.java

```diff
@@ -560,17 +561,17 @@ private void _saveCommit( String fileName, String outputFileName, boolean saveVe
         
         if ( prettyFormat ) {
             String formatCmd = new String(System.getProperty("uvm.bin.dir") + "/" + "ut-format-json" + " " + outputFileName );
-            UvmContextImpl.context().execManager().setLevel(org.apache.log4j.Level.DEBUG);
+            UvmContextImpl.context().execManager().setLevel(org.apache.logging.log4j.Level.DEBUG);
```

Comment:
Better to import `org.apache.logging.log4j.Level` instead of putting entire package name here on and on other lines

Reply:
Modified all files using fully qualified class name of Level class